src/components/car/CarComponent.tsx

```diff
@@ -37,12 +39,15 @@ export default class CarComponent extends React.Component<Props, State> {
     const carUsers = car?.carUsers ?? [];
     const defaultCarUser = carUsers.length === 1 ? carUsers[0] : carUsers?.find((carUser) => carUser.default === true);
     const defaultCarUserName = Utils.buildUserName(defaultCarUser?.user);
+    const isNameHyphen = defaultCarUserName === Constants.HYPHEN;
     const otherUserCount = Math.max(carUsers.length - 1, 0);
     const carCatalog = car?.carCatalog;
     const carMake = carCatalog?.vehicleMake ?? '';
     const carModel = carCatalog?.vehicleModel ?? '';
     const carModelVersion = carCatalog?.vehicleModelVersion ?? '';
     const carFullName = carMake + ' ' + carModel + ' ' + carModelVersion;
+    const carLicensePlate = car.licensePlate;
+    const userIDs = carUsers.map((userCar: UserCar) => userCar?.user?.id).join('|');
```

Comment:
Let the userIDs as an Array of IDs the join will be used when calling the API in User list.